source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -648,42 +650,46 @@ see the following API Documentation:
 - `TypeWithTypeParameters <{+api+}/apidocs/bson/org/bson/codecs/pojo/TypeWithTypeParameters.html>`__
 
 For more information on generics and type parameters, see the
-:`Oracle documentation on Invoking and Instantiating a Generic Type <https://docs.oracle.com/javase/tutorial/java/generics/types.html#instantiation>`__.
+`Java language guide on Invoking and Instantiating a Generic Type <https://docs.oracle.com/javase/tutorial/java/generics/types.html#instantiation>`__.
 
 Enum Type Support
 ^^^^^^^^^^^^^^^^^
 
-The POJO ``Codec`` uses the ``enum`` constant name as the property value by
-default. The following code snippet shows a POJO that contains an ``enum``
-field:
-
-.. code-block:: java
+The ``PojoCodecProvder`` includes a codec to convert the ``enum`` property
+type field in your POJO. The codec uses the name of the ``enum`` as the
+property value by default.
 
-   public enum Family {
-       ADELIE,
-       EMPEROR,
-       GENTOO,
-       ROCKHOPPER
-   }
+.. note::
 
-   public class Penguin {
+   When using driver versions 4.5 and later, you can include the ``EnumCodec``
+   without the ``PojoCodecProvider``.
```

Comment:
Should this be shown in the code below?

Reply:
No, this page is generally about using the PojoCodecProvider. The admonition informs of a different case (working without the PojoCodecProvider) that is tangentially related. Since this is the only content that relates to enums and PojoCodecProvider, I wasn't able to find any other suitable location.